runelite-client/src/main/java/net/runelite/client/plugins/statusorbs/StatusOrbsPlugin.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2018 TheStonedTurtle <https://github.com/TheStonedTurtle>
+ * Regen Meter Contributors:
```

Comment:
These contributor entries should (a) not be separated by type, and (b) not all added in this commit; they should be added in the later commits where the other plugins are merged into this one.